sql/analyzer/resolve_columns_test.go

```diff
@@ -75,6 +75,64 @@ func TestMisusedAlias(t *testing.T) {
 	require.EqualError(err, ErrMisusedAlias.New("alias_i").Error())
 }
 
+func TestNoTuplesProjected(t *testing.T) {
+	require := require.New(t)
+	f := getRule("no_tuples_projected")
+
+	table := mem.NewTable("mytable", sql.Schema{
+		{Name: "i", Type: sql.Int32},
+	})
+
+	node := plan.NewProject([]sql.Expression{
+		expression.NewTuple(
```

Comment:
We should add a new test case in which this `expression.NewTuple` is wrapped in a `expression.NewAlias`, for example. With that case, the current implementation would fail, but I already suggested a fix to that in my previous comment.

Reply:
Done (in the testcase for the existing rule).